sdk/database/helper/connutil/sql.go

```diff
@@ -40,6 +40,7 @@ type SQLConnectionProducer struct {
 	AuthType                 string      `json:"auth_type" mapstructure:"auth_type" structs:"auth_type"`
 	ServiceAccountJSON       string      `json:"service_account_json" mapstructure:"service_account_json" structs:"service_account_json"`
 	DisableEscaping          bool        `json:"disable_escaping" mapstructure:"disable_escaping" structs:"disable_escaping"`
+	usePrivateIp             bool        `json:"use_private_ip" mapstructure:"use_private_ip" structs:"use_private_ip"`
```

Comment:
Nit: Rename this to be `usePrivateIP` to keep the acronym in capital letters to match others?

Reply:
fixed !